### PR TITLE
fixed 6 of the remaining 17 html Validation errors

### PR DESCRIPTION
**Pull request by thirlz95**

### CODE REVIEW

File: app/uk/gov/hmrc/nisp/views/main.scala.html [hunk 1]
```diff
@@ -209,7 +209,7 @@ <h1 class=@headerClass> @Html(title) </h1>
     if (mainTemplateParams.h1Class.isDefined){
         mainTemplateParams.h1Class.get
     } else {
-        "heading-large top-title-heading heading-bottom-margin"
+        mainTemplateParams.h1Class.getOrElse("heading-large top-title-heading heading-bottom-margin")
```
Comment (frikit) [inline app/uk/gov/hmrc/nisp/views/main.scala.html]: You can remove whole if and put only
 
mainTemplateParams.h1Class.getOrElse("heading-large top-title-heading heading-bottom-margin")

Comment (frikit) [reply app/uk/gov/hmrc/nisp/views/main.scala.html]: Will look like:
 @headerClass = @{
    mainTemplateParams.h1Class.getOrElse("heading-large top-title-heading heading-bottom-margin")
}

Comment (thirlz95) [reply app/uk/gov/hmrc/nisp/views/main.scala.html]: cheers mate, i'll make the changes now to the templates

**Comment by frikit** [comment]
Looks good

**Pull request merged**